runtime-modules/content/src/lib.rs

```diff
@@ -492,7 +494,7 @@ decl_module! {
                 collaborators: params.collaborators.clone(),
                 cumulative_reward_claimed: BalanceOf::<T>::zero(),
                 transfer_status: ChannelTransferStatus::NoActiveTransfer,
-                privilege_level: Zero::zero(),
+                privilege_level: Zero::zero(), // start with lowes privilege level
```

Comment:
```suggestion
                privilege_level: Zero::zero(),
```
The default `privilege_level` (`0`) is not necessarily the lowest, by itself it's just a meaningless number, it's up to the content working group leader to define how given level corresponds to curator groups' permissions.